flink-connector-kudu/src/main/java/org/apache/flink/connector/kudu/connector/writer/KuduWriter.java

```diff
@@ -35,11 +36,12 @@
 
 import java.io.IOException;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 
 /** Writer to write data to a Kudu table. */
 @Internal
-public class KuduWriter<T> implements AutoCloseable {
+public class KuduWriter<T> implements SinkWriter<T>, AutoCloseable {
```

Comment:
`AutoClosable` is redundant and can be removed

Reply:
Good catch!